marlowe/src/Language/Marlowe/Core/V1/Semantics.hs

```diff
@@ -1,3 +1,5 @@
+-- editorconfig-checker-disable-file
```

Comment:
Should we remove this ? now that we disabled the feature ?

Reply:
Yes, can remove

Reply:
Fixed in 34f1d7e5d89e68996254f165386a3264372c3252.